gem/lib/rbi-central/index.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module RBICentral
+  INDEX_PATH = "index.json"
```

Comment:
Do we need to change this to something like `../index.json` or move `index.json` under `gem`?

Reply:
No the commands are meant to be run from the repo root directory through `bundle exec <command>` so the index is going to be at the same path.